srcpkgs/calibre/template

```diff
@@ -21,7 +21,7 @@ maintainer="Orphaned <orphan@voidlinux.eu>"
 license="GPL-3"
```

Comment:
please use spdx identifier for the license

Reply:
Corrected the license to SPDX format

Reply:
Source on the license being GPL-3.0-only: https://www.mobileread.com/forums/showthread.php?s=673810cac0038765ef1402e695172891&t=313261